express-api/src/routes/buildingsRouter.ts

```diff
@@ -0,0 +1,19 @@
+import controllers from '@/controllers';
+import express from 'express';
+
+export const BUILDINGS_ROUTE = '/properties/buildings';
+
+const router = express.Router();
+
+// Endpoints for buildings data manipulation
+router.route(`${BUILDINGS_ROUTE}/:buildingId`).get(controllers.getBuilding);
+router.route(`${BUILDINGS_ROUTE}/:buildingId`).put(controllers.updateBuilding);
+router.route(`${BUILDINGS_ROUTE}/:buildingId`).delete(controllers.deleteBuilding);
+router.route(BUILDINGS_ROUTE).get(controllers.filterBuildingsQueryString);
+router.route(BUILDINGS_ROUTE).post(controllers.addBuilding);
```

Comment:
You can actually chain together http method calls if they use the same route:
```suggestion
router.route(`${BUILDINGS_ROUTE}/:buildingId`)
  .get(controllers.getBuilding)
  .put(controllers.updateBuilding)
  .delete(controllers.deleteBuilding);
router.route(BUILDINGS_ROUTE)
  .get(controllers.filterBuildingsQueryString)
  .post(controllers.addBuilding);
```